### PR TITLE
SM-7061 - Permit duration challenge review endpoint

### PR DESCRIPTION
**Pull request by e-mccoy**

### CODE REVIEW

File: docs/api-documentation/V3/V3.9/json/work-swagger.json [hunk 1]
```diff
@@ -432,6 +432,22 @@
 				"type": "object",
 				"additionalProperties": false
 			},
+			"DurationChallengeReviewStatusResponse": {
```
Comment (cpaul92) [inline docs/api-documentation/V3/V3.9/json/work-swagger.json:435]: No big deal but you don't need to update this now, will be updated as part of release

File: docs/api-documentation/versions-and-changes/v3/changelog.md [hunk 2]
```diff
@@ -15,6 +15,21 @@ Updated Work API with the following changes:
 {: .govuk-body}
 <ol class="govuk-list govuk-list--bullet">
   <li>SM-6738: Updated the <code>GET /works/{workReferenceNumber}/sites/{siteNumber}</code> endpoint to limit the number of reinstatements returned from a site response to the latest 50.</li>
+  <li>
+    SM-7061: Added <code>PUT /works/{workReferenceNumber}/permits/{permitReferenceNumber}/duration-challenge-review</code> endpoint to allow promoters to submit duration challenge reviews on immediate permits. The following fields are also now returned in the <code>PermitResponse</code>:
```
Comment (cpaul92) [inline docs/api-documentation/versions-and-changes/v3/changelog.md:19]: Also need an update to V2 to list the new fields returned from PermitResponse there too

**Review by ccurran01 — COMMENTED** [review]
Couple of additional places it'd be worth keeping up to date:

* the security section has a table of permissions per endpoint, the new one should be added there for V3
* There is a duration challenge non-acceptance header in the v3 resource guide. Something should be added there also.

File: docs/_includes/api-documentation/v3/resource-guide.md [hunk 3]
```diff
@@ -939,14 +939,15 @@ The available statuses are -
...
 {: .govuk-body}
 
 The <code>PUT works/{workReferenceNumber}/permits/{permitReferenceNumber}/alterations/{permitAlterationReferenceNumber}/duration-challenge-non-acceptance-response</code> endpoint will allow the Highway Authority user to respond to a duration challenge review requested by a promoter. The Highway Authority can choose to update the reasonable period end date, or to keep the date as submitted when providing the original duration challenge details. Relevant fields are returned in the PermitAlterationResponse.
 {: .govuk-body}
 
+The <code>PUT works/{workReferenceNumber}/permits/{permitReferenceNumber}/duration-challenge-review</code> endpoint will allow the Planner to accept or not accept a duration challenge on an immediate permit; if the Planner does not accept the duration challenge, a reason must be given. Relevant fields are returned in the PermitAlterationResponse.
```
Comment (ccurran01) [inline docs/_includes/api-documentation/v3/resource-guide.md:948]: Should this be permit response? 

```suggestion
The <code>PUT works/{workReferenceNumber}/permits/{permitReferenceNumber}/duration-challenge-review</code> endpoint will allow the Planner to accept or not accept a duration challenge on an immediate permit; if the Planner does not accept the duration challenge, a reason must be given. Relevant fields are returned in the PermitResponse.
```

**Review by cpaul92 — APPROVED** [review]
Approved pending addressing Cormac's comment

**Review by ccurran01 — APPROVED** [review]

**Pull request merged**